src/main/g8/.scalafix.conf

```diff
@@ -1,12 +1,14 @@
 rules = [
   ProcedureSyntax,
-  RemoveUnused,
```

Comment:
This Scalafix built-in [may conflict with the `removeUnused` option in `OrganizeImports`](https://github.com/liancheng/scalafix-organize-imports/#1-getting-started). The recommendation in `organize-imports` is to rely on the `organize-imports` implementation instead. I could go either way on this, but I lean towards using `organize-imports` to keep all of our import managing limited to one package.